.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "🐞 Bug report"
+about: Something is not working as it should
+
+---
+
+#### Actual behavior
+
+...
+
+#### Expected behavior
+
+...
+
+#### Code to reproduce
+
+```js
+...
+```
```

Comment:
We should also encourage submitting a PR with a failing test in addition. This will make it more likely for us to prioritize your issue and it's a good way to prove that the issue is with Got and not your code.

And link to https://github.com/avajs/ava#failing-tests